apps/artemis-web/utils/graphql.ts

```diff
@@ -23,17 +23,26 @@ const requestToken = async () => {
 let apolloClient: ApolloClient<NormalizedCacheObject>;
 
 const createApolloClient = (GRAPHQL_URI, GRAPHQL_WS_URI) => {
-  const httpLink = createHttpLink({
-    uri: GRAPHQL_URI,
-    useGETForQueries: false,
-    headers: async () => {
-      await requestToken();
-      return {
-        authorization: `Bearer ${accessToken}`,
-        // 'x-hasura-admin-secret': process.env.HASURA_SECRET
-      };
-    },
-  });
+  const httpLink =
```

Comment:
why was the previous one not working? did you test it with queries and it was broken?

Reply:
yeap it was broken...